tailwind.config.js

```diff
@@ -9,13 +9,15 @@ module.exports = {
       boxShadow: {
         fb: '0 -2px 5px rgb(128,128,128)',
         fbpanel: '0 1px 10px grey',
+        img: '2px 3px 12px #d3d3d3',
       },
       // compiled the css with the below primary and secondary colors and linked with _app.tsx.
       colors: {
         primary: '#C54582',
         secondary: '#F4DCF4',
         'app-background': '#fff',
         orange: '#ff5a42',
+        lightGray: '#8a8888',
```

Comment:
enter purpose instead of color name/ e.g. shadow or border

Reply:
It is used across many component's , I'm not sure what purpose It is for .
It is used in heading and content as well.

Reply:
the lightGray is defined in this PR itself. So I guess the usage will be limited to this PR